pages/api/submit.ts

```diff
@@ -0,0 +1,60 @@
+/* eslint camelcase:0, new-cap:0 */
+import Daet from 'daet'
+import { NextApiRequest, NextApiResponse } from 'next'
+
+/* Gets the data from client side when user creates a new event
+ * by filling up a Form and clicking Submit. Here we should
+ * create the faundaDb entry for the same. */
+export default async function sendEvents(
+	req: NextApiRequest,
+	res: NextApiResponse
+) {
+	const faunadb = require('faunadb')
+	const q = faunadb.query
+	const faunaEnv = JSON.parse(process.env.FAUNADB || '{ }')
+	const faunaSecret = faunaEnv.FAUNADB_SECRET_KEY
```

Comment:
make above use the same `FAUNA_SERVER_KEY`, which should be fetch from the config file